go-controller/cmd/ovn-k8s-cni-overlay/app/helper_windows.go

```diff
@@ -145,6 +145,8 @@ func ConfigureInterface(args *skel.CmdArgs, namespace string, podName string, ma
 			IPAddress:      ipAddr,
 			MacAddress:     macAddressIpFormat,
 			PrefixLength:   uint8(ipMaskSize),
+			DNSServerList:  strings.Join(conf.NetConf.DNS.Nameservers, ","),
```

Comment:
Eventually we will want to write the DNS config through ovnkube config flags. But we can hold off on that one until we know what the long term fix is going to be.